tutorials/fundamentals/trim.md

```diff
@@ -0,0 +1,26 @@
+To trim characters from a string in JavaScript, you must utilize the `replace()` function.
+`trim()` is only useful for removing whitespace characters which is why `replace()` needs to be used instead.
+`replace()` takes two arguments:
+- a regular expression denoting what to take out
+- a string denoting what to put in.
```

Comment:
remove the last `.`

Better to consistently end all elements in a list with `.` or consistently avoid `.`